processing/src/main/java/io/druid/segment/incremental/OffheapIncrementalIndex.java

```diff
@@ -92,6 +93,26 @@ public OffheapIncrementalIndex(
   }
 
   public OffheapIncrementalIndex(
```

Comment:
There are no compatibility concerns, so suggested to inline all usages of this constructor and remove it.